core/includes/common.inc

```diff
@@ -3649,7 +3649,7 @@ function backdrop_group_css($css) {
  * @see system_element_info()
  */
 function backdrop_aggregate_css(&$css_groups) {
-  $preprocess_css = !defined('MAINTENANCE_MODE') && config_get('system.core', 'preprocess_css');
+  $preprocess_css = !defined('MAINTENANCE_MODE') && settings_get('preprocess_css', config_get('system.core', 'preprocess_css'));
```

Comment:
Can we please add a comment above all of these lines? Like so:

`// Try retrieving the setting from settings.php first; fall back to config.`

Reply:
I'm not sure the comment adds anything here. But I do agree that in general we want to make it clear that certain config can be overridden. Again, this might be better in the follow-up.